components/__init__.py

```diff
@@ -247,6 +244,16 @@ def define_components(reqs):
                 commands=isal_build,
                 required_progs=['nasm', 'yasm'],
                 libs=["isal"])
+    reqs.define('isal_crypto',
+                retriever=GitRepoRetriever(
+                        "https://github.com/intel/isa-l_crypto"),
+                commands=['./autogen.sh ',
+                          './configure --prefix=$ISAL_CRYPTO_PREFIX '
+                          '--libdir=$ISAL_CRYPTO_PREFIX/lib',
+                          'make $JOBS_OPT', 'make install'],
+                required_progs=['nasm', 'yasm'],
+                libs=['isal_crypto'],
+                )
```

Comment:
(pylint-bad-continuation) Wrong continued indentation (remove 1 space).

Reply:
the parenthesis is wrong.

Reply:
fix will be in next patch